desktop/src/main/java/bisq/desktop/primary/main/content/trade/bisqEasy/chat/BisqEasyChatController.java

```diff
@@ -281,4 +269,15 @@ void onCompleteTrade() {
     boolean isMyMessage(ChatMessage chatMessage) {
         return userIdentityService.isUserIdentityPresent(chatMessage.getAuthorId());
     }
+
+    boolean isBtcSeller() {
+        checkArgument(lastOfferMessage.getTradeChatOffer().isPresent());
+        var offer = lastOfferMessage.getTradeChatOffer().get();
```

Comment:
We could use here also `.orThrow` if the value need to be present.

Reply:
I prefer check or assert to indicate there is a bug if the check triggers. Is there a convention in the project to use throw here?